java/org/apache/catalina/core/StandardContextValve.java

```diff
@@ -81,7 +82,17 @@ public final void invoke(Request request, Response response)
 
         // Acknowledge the request
         try {
-            response.sendAcknowledgement();
+            // Acknowledge based on the policy
+            final ContinueHandlingResponsePolicy continueHandlingResponsePolicy = (ContinueHandlingResponsePolicy) request.getConnector().getProperty("continueHandlingResponsePolicy");
+
```

Comment:
Code lines need to wrap at 120 chars. This one looks too long.

Reply:
thanks, fixed